res/TensorFlowLiteRecipes/Net_Conv_Mul_000/test.recipe

```diff
@@ -0,0 +1,71 @@
+# test for Conv(pad=valid, krnl(8, 3, 2, 4), stride=(2, 1))->Mul_with_channelwise_constant
```

Comment:
Q) does pad difference concern conv-mul fusion?

Reply:
all kernel numbers < 10 but can we have numbers like 2 or 3?
currently there are lots of value tests and it's getting huge and time consuming.

Reply:
> Q) does pad difference concern conv-mul fusion?

No. No difference. Just different parameter to make sure it's irrelevant for value test. Should i remove it?

> all kernel numbers < 10 but can we have numbers like 2 or 3? currently there are lots of value tests and it's getting huge and time consuming.

@seanshpark 
tests used `stride=2` and `kernel_height=3`, so minimal value for `h` is 4, Is it ok to use 4? or should i change `kernel_height` or `stride`?

Reply:
> Should i remove it?

If it's irrelevant , I think we don't have to add them

>  Is it ok to use 4? 

4 seems OK :)

Reply:
> If it's irrelevant , I think we don't have to add them
> 4 seems OK :)

Ok. Thank you. I'll change tests.

Reply:
done